docs/api/Model.md

```diff
@@ -173,7 +173,7 @@ User.get(1, {"return": "request"}, (error, request) => {
 });
 ```
 
-In the event you have a rangeKey for your model, you can pass in an object for the `hashKey` parameter.
+In the event you have a rangeKey (sort key) for your model, you can pass in an object for the `hashKey` (partition key) parameter.
```

Comment:
I think this should read something like the following.

```suggestion
In the event you have a rangeKey/sort key for your model, you can pass in an object for the `key` parameter which includes the range key/sort key & the hash key/partition key.
```

Then line 127 should be changed to the suggestion above.

Reply:
> Few suggestions. Still not sure I'm on board about the `Item` vs `Document` thing yet. I think that should maybe be a separate PR and we rename every instance of it.

Yeah lets do that in a separate step!